packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -137,6 +139,23 @@ bool GetValueFromEncodableMap(const flutter::EncodableValue& arguments,
   return false;
 }
 
+bool needsSWBackend(void) {
+  bool result = false;
+  char* value;
+  int ret;
+  ret = system_info_get_platform_string("http://tizen.org/system/model_name",
+                                        &value);
+  if (ret == SYSTEM_INFO_ERROR_NONE) {
+    if (strcmp(value, "Emulator") == 0) {
+      result = true;
+    }
+  }
```

Comment:
```suggestion
  if ((SYSTEM_INFO_ERROR_NONE == ret) && (0 == strcmp(value, "Emulator")) {
      result = true;
  }
```
How about reducing the number of nested if statements? And how about putting the value in front of the variable in the conditional statement?

Reply:
Thanks! That looks good! 👍